code/game/objects/items/knives.dm

```diff
@@ -156,6 +156,49 @@
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0)
 	custom_materials = list(/datum/material/glass=400)
 
+/obj/item/knife/shiv/plasma
+	name = "plasma shiv"
+	icon = 'icons/obj/shards.dmi'
+	icon_state = "plasmashiv"
+	inhand_icon_state = "plasmashiv"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	desc = "A makeshift plasma glass shiv."
+	force = 9
+	throwforce = 13
+	armor = list(MELEE = 25, BULLET = 25, LASER = 25, ENERGY = 25, BOMB = 25, BIO = 0, FIRE = 50, ACID = 50)
+	custom_materials = list(/datum/material/glass=400, /datum/material/plasma=200)
+
+/obj/item/knife/shiv/titanium
+	name = "titanium shiv"
+	icon = 'icons/obj/shards.dmi'
+	icon_state = "titaniumshiv"
+	inhand_icon_state = "titaniumshiv"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	desc = "A makeshift titanium infused glass shiv."
+	force = 8
```

Comment:
Isn't it redundant to have `force = 8` here? It inherits the exact same damage from the base shiv, no? Same stuff for the following in all three of them:
`icon = 'icons/obj/shards.dmi'`
`lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'`
`righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'`